language/compiler/src/unit_tests/import_tests.rs

```diff
@@ -1,63 +0,0 @@
-// Copyright (c) The Diem Core Contributors
```

Comment:
I'm deleting this test since the property it's supposed to test (being able to compile scripts/modules with imports) is covered by many other tests.